popSimManu.tex

```diff
@@ -557,10 +583,11 @@ \section*{Discussion}
 
 Here we have described the first major product from the PopSim Consortium:
 the \stdpopsim library. We have founded the Consortium with a number of specific goals in mind:
-standardization of simulation within the computational genetics community,
+standardization of simulation within the population genetics community,
 increasing reproducibility and ease of use of complex simulation based results,
 community-based development and decision making guiding best practices in population genetics,
 and eventual benchmarking of inferential methods.
+% ALG: Should these goals be mentioned in the intro?
```

Comment:
I think it does in other words, and we don't want to completely repeat ourselves.